configs/defs/generic_linux.textproto

```diff
@@ -9558,3 +9558,1301 @@ benchmark_configs: {
       "}"
   }
 }
+benchmark_configs: {
+  id: "separate-partition-home"
+  compliance_note: {
+    title: "Ensure separate partition exists for /home"
```

Comment:
Same as in the previous PR, some of these checks are already defined in other textprotos:

https://github.com/google/localtoast/blob/16cb3b302fa5155317c449c8d6a25b98ace56e91/configs/defs/rocky.textproto#L1757

So if these now apply to more linux OSes we should move them around as appropriate and indicate with the version fields what they apply to